src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/version/GerritVersionChecker.java

```diff
@@ -66,7 +66,17 @@
         /**
          * Gerrit CommentAdded always contains approval information, added in Gerrit 2.13.
          */
-        commentAlwaysApproval("CommentAdded always contains approval", "2.13");
+        commentAlwaysApproval("CommentAdded always contains approval", "2.13"),
+        
+        /**
+         * WIPStateChanged events, added in Gerrit 2.15.
+         */
+        wipStateChanged("WIPStateChanged events", "2.15"),
+        
+        /**
+         * PrivateStateChanged events, added in Gerrit 2.15.
+         */
+        privateStateChanged("privateStateChanged events", "2.15");
```

Comment:
You will have to check if these features are enabled before using them, it's not enough to add them as Features.